microcosm_postgres/operations.py

```diff
@@ -2,9 +2,18 @@
 Common database operations.
 
 """
+from microcosm_postgres.migrate import main
 from microcosm_postgres.models import Model
 
 
+def stamp_head(graph):
```

Comment:
:+1:  I think being able to stamp head and/or apply all migrations via the scripts should be something that is easy to do and does not require user knowledge of migration dir location.